frontend/docs/css-style.guide.md

```diff
@@ -0,0 +1,277 @@
+
+    $cornflower-blue: #4285F4;
+
+Color name should describe the color as it is and not where is used.
+
+Avoid:
+
+    background-color: $table-header-color;
+
+Avoid:
+
+    color: $main-title;
+
+Instead, use:
+
+    color: $cornflower-blue;
+
+To avoid color repetition (Ej: _$light-grey_, _$lighter-grey_) we use [Name that color](http://chir.ag/projects/name-that-color/).
+
+Use color abbreviations if possible, changing
+
+    $beauty-bush: #EEBBCC;
+
+for
+
+    $beauty-bush: #EBC;
+
+Color hex should not be in lowercase, changing
+
+    $beauty-bush: #ebc;
+
+for
+
+    $beauty-bush: #EBC;
+
+#### Properties order:
+Properties must be alphabetically ordered
+
+    background: fuchsia;
+    border: 1px solid;
+    border-radius: 4px;
+    color: black;
+    text-align: center;
+    text-indent: 2em;
+
+#### Indentation:
+Property should be indented with 2 spaces respect from its containing block
+
+    .test {
+      display: block;
+      height: 100px;
+    }
+
+In case of nested blocks, they should be separed with one blank link  and indented with 2 spaces
+
+    .test {
+
+      .caption {
+        color: $cornflower-blue;
+      }
+    }
+
+#### Declaration stop:
+Declarations should end with “;”
+
+    .test {
+      display: block;
+      height: 100px;
+    }
+
+Avoid lines with no declarations stop such as:
+
+    .test {
+      display: block;
+      height: 100px
+    }
+
+#### Propery names:
+Property names should be followed by one space
+
+    display:block; //Wrong
+    display: block; //Right
+
+#### Block names:
+Block names should be followed by one space
+
+    .video{  //Wrong
+      margin-top: 10px;
+    }
+
+    .video
+    {  //Wrong
+      margin-top: 10px;
+    }
+
+    .video {  //Right
+      margin-top: 10px;
+    }
+
+#### Multiple selectors per block:
+When many selectos has the same properties, it is recommended to use a single block with multiple selectors
+
+    .sale-option:focus,
+    .sale-option:active {
+      color: $green;
+    }
+
+Multiple selectors must not be written in the same line
+
+    .sale-option:focus, .sale-option:active {  //Incorrecto
+      color: $green;
+    }
+
+#### Rules separation
+Rules must be separated with a blank line
+
+    .container {
+      background: $white;
+    }
+    .tile { // No blank line, wrong
+      margin: auto;
+      width: 50%;
+    }
+
+    .container {
+      background: $white;
+    }
+
+    .tile { // Blank line, right
+      margin: auto;
+      width: 50%;
+    }
+
+
+#### Quotes
+Use single quotes
+
+    .caption {
+      font-family: “open sans”, arial, sans-serif; // Wrong
+    }
+
+    .caption {
+      font-family: 'open sans', arial, sans-serif; // Right
+    }
+
+##  Naming:
+There are many different approaches to css naming (expressive, atomic, object-oriented). The most important thing (in case of maintaining existing css) is to create classes that matches the style and approach already implemented.
```

Comment:
I would erase the part of "and approach already implemented", mainly because I don't think it is grammatically correct (the object of the approach is missing) and I don't think it adds much to previous idea (matching the style). Maybe you can just say "... that match already implemented style" and avoid the rest.